source/includes/options/option-mongosqld-schemaName.rst

```diff
@@ -28,8 +28,13 @@
           <mongosqld --schemaSource>` database after the |bi-short|
           samples the schema on startup.
    
-   If not specified, defaults to ``defaultSchema``.
-   
+   If you upload a custom schema, you must give it a name with
```

Comment:
Can we get this in a "Note" or "Important" section?

Reply:
Done